cmd/kubestellar-where-resolver/cmd/where-resolver-cmd.go

```diff
@@ -83,39 +92,43 @@ func Run(ctx context.Context, options *resolveroptions.Options) error {
 	logger := klog.Background()
 	ctx = klog.NewContext(ctx, logger)
 
-	// create edgeSharedInformerFactory
-	espwRestConfig, err := options.EspwClientOpts.ToRESTConfig()
+	spaceManagementConfig, err := options.SpaceMgtOpts.ToRESTConfig()
 	if err != nil {
-		logger.Error(err, "failed to create config from flags")
+		logger.Error(err, "Failed to create space management config from flags")
```

Comment:
Should be something clearer about the config, like one of the following.

- "Failed to create space management API client config from flags"
- "Failed to create space management API space client config from flags"

Reply:
Changed to "Failed to create space management API client config from flags"